examples/scripts/deconv_modl_train_foam1.py

```diff
@@ -72,7 +73,7 @@
 
 n = 3  # convolution kernel size
 σ = 20.0 / 255  # noise level
-psf = jnp.ones((n, n)) / (n * n)  # blur kernel
+psf = np.ones((n, n)) / (n * n)  # blur kernel
```

Comment:
Why the general shift from jax to numpy arrays?

Reply:
Mainly to benefit from the mutability of numpy arrays on other parts of the script. It seems also useful to indicate elements that will be regarded as constant in the script.